### DISCUSSION TITLE
Let's encrypt renovation error

### DISCUSSION THREAD

**Discussion by rojano9** [globaleaks/globaleaks-whistleblowing-software#3583] [Q&A]
Hi,

I was trying to renew the Let's Encrypt certificate, but I encountered an error:
"Error!
Internal Server Error (Unexpected)"

If I check the globaleaks.log, it gives me the following error: Exception mail suppressed for exception (<class 'acme.errors.ValidationError'>) [reason: special exception]

My version of GlobaLeaks is 4.12.8. I've noticed in the changelog that version 4.12.9 has been released, which states that an ACME error is fixed. However, when trying to update using the command apt-get update && apt-get install globaleaks, it indicates that I'm on the latest available version.

Thanks,

**Comment by rojano9** [comment]
After trying a few more times, I have received the following email with errors: 

AttributeError Attribute not found.

Traceback (most recent call last):

  File "/usr/lib/python3/dist-packages/twisted/internet/defer.py", line 1418, in _inlineCallbacks
    result = g.send(result)

  File "/usr/lib/python3/dist-packages/globaleaks/rest/api.py", line 425, in concludeHandlerFailure
    self.handle_exception(err, request)

  File "/usr/lib/python3/dist-packages/globaleaks/rest/api.py", line 252, in handle_exception
    e.tid = request.tid

  File "/usr/lib/python3/dist-packages/josepy/util.py", line 179, in __setattr__
    raise AttributeError("can't set attribute")

AttributeError: can't set attribute

**Comment by evilaliv3** [comment]
Thank you @rojano9 

Which distribution / version are you using?

**Reply by rojano9** [reply]
Im using Ubuntu 20.04.4 LTS focal

**Comment by evilaliv3** [comment]
I suspect on Focal, for which by mistake we forgot to issue the release 4.12.9.

We will fix this issue by today.

**Reply by rojano9** [reply]
Thank you so much.

**Reply by rojano9** [reply]
Hi,

I have updated my version of Globaleaks to version 4.12.9, but I still encounter the same error when I try to renew the certificate.

**Comment by evilaliv3** [comment]
Thank you @rojano9 

for the moment i think the best for your case is to reset the certificate and issue a new one.

Would you please try if this works?

In the meantime i'm trying to check the error because i noticed that there is an exception inside the exception causing to not being able to read the original one.

**Reply by rojano9** [reply]
I clicked on reset and then tried to regenerate the certificate, keep getting the same error?
Or do you mean generating a new certificate for a new domain?

**Reply by rojano9** [reply]
I have finally generated a certificate externally and added it manually. This way, there are no issues.
Thanks,

**Reply by TomFuch** [reply]
The same Error occurs on Debian 11.
